diff --git a/node_modules/@polkadot/rpc-augment/augment/jsonrpc.d.ts b/node_modules/@polkadot/rpc-augment/augment/jsonrpc.d.ts
index 588d778..3cc2aea 100644
--- a/node_modules/@polkadot/rpc-augment/augment/jsonrpc.d.ts
+++ b/node_modules/@polkadot/rpc-augment/augment/jsonrpc.d.ts
@@ -429,11 +429,11 @@ declare module '@polkadot/rpc-core/types/jsonrpc' {
             /**
              * Generate MMR proof for the given leaf indices.
              **/
-            generateBatchProof: AugmentedRpc<(leafIndices: Vec<u64> | (u64 | AnyNumber | Uint8Array)[], at?: BlockHash | string | Uint8Array) => Observable<MmrLeafProof>>;
+            generateBatchProof: AugmentedRpc<(leafIndices: Vec<u64> | (u64 | AnyNumber | Uint8Array)[], at?: BlockHash | string | Uint8Array) => Observable<MmrLeafiBatchProof>>;
             /**
              * Generate MMR proof for given leaf index.
              **/
-            generateProof: AugmentedRpc<(leafIndex: u64 | AnyNumber | Uint8Array, at?: BlockHash | string | Uint8Array) => Observable<MmrLeafBatchProof>>;
+            generateProof: AugmentedRpc<(leafIndex: u64 | AnyNumber | Uint8Array, at?: BlockHash | string | Uint8Array) => Observable<MmrLeafProof>>;
         };
         net: {
             /**
