diff --git a/node_modules/@polkadot/types/bundle-polkadot-types.js b/node_modules/@polkadot/types/bundle-polkadot-types.js
index b0d11bb..4b1e9fd 100644
--- a/node_modules/@polkadot/types/bundle-polkadot-types.js
+++ b/node_modules/@polkadot/types/bundle-polkadot-types.js
@@ -2795,7 +2795,7 @@
         name: 'at',
         type: 'BlockHash'
       }],
-      type: 'MmrLeafProof'
+      type: 'MmrLeafBatchProof'
     },
     generateProof: {
       description: 'Generate MMR proof for given leaf index.',
@@ -2808,7 +2808,7 @@
         name: 'at',
         type: 'BlockHash'
       }],
-      type: 'MmrLeafBatchProof'
+      type: 'MmrLeafProof'
     }
   };
 
diff --git a/node_modules/@polkadot/types/cjs/interfaces/mmr/rpc.js b/node_modules/@polkadot/types/cjs/interfaces/mmr/rpc.js
index a8e8c9a..714394e 100644
--- a/node_modules/@polkadot/types/cjs/interfaces/mmr/rpc.js
+++ b/node_modules/@polkadot/types/cjs/interfaces/mmr/rpc.js
@@ -19,7 +19,7 @@ const rpc = {
       name: 'at',
       type: 'BlockHash'
     }],
-    type: 'MmrLeafProof'
+    type: 'MmrLeafBatchProof'
   },
   generateProof: {
     description: 'Generate MMR proof for given leaf index.',
@@ -32,7 +32,7 @@ const rpc = {
       name: 'at',
       type: 'BlockHash'
     }],
-    type: 'MmrLeafBatchProof'
+    type: 'MmrLeafProof'
   }
 };
 exports.rpc = rpc;
\ No newline at end of file
diff --git a/node_modules/@polkadot/types/interfaces/mmr/rpc.js b/node_modules/@polkadot/types/interfaces/mmr/rpc.js
index 9dcfab2..4282093 100644
--- a/node_modules/@polkadot/types/interfaces/mmr/rpc.js
+++ b/node_modules/@polkadot/types/interfaces/mmr/rpc.js
@@ -13,7 +13,7 @@ export const rpc = {
       name: 'at',
       type: 'BlockHash'
     }],
-    type: 'MmrLeafProof'
+    type: 'MmrLeafBatchProof'
   },
   generateProof: {
     description: 'Generate MMR proof for given leaf index.',
@@ -26,6 +26,6 @@ export const rpc = {
       name: 'at',
       type: 'BlockHash'
     }],
-    type: 'MmrLeafBatchProof'
+    type: 'MmrLeafProof'
   }
 };
